nexus_constructor/qentity_utils.py

```diff
@@ -0,0 +1,41 @@
+from typing import List
+
+from PySide2.Qt3DCore import Qt3DCore
+from PySide2.Qt3DRender import Qt3DRender
+from PySide2.QtGui import QColor
+
+
+def set_material_properties(
```

Comment:
I think that in almost all the places you call this you basically do:

```
material = Qt3DExtras.QPhongMaterial()
set_material_properties(material)
```
Maybe it's worth moving the material creation into this method and streamline the code elsewhere.